Makefile

```diff
@@ -11,17 +11,17 @@ RM := --rm
 GO_FLAGS := -ldflags "-extldflags \"-static\" -linkmode=external -s -w" -tags netgo -i
 
 # A 3-year-old image which is a reasonable match for the last time the files were generated.
-PROTOC_IMAGE=namely/protoc:1.22_1
+PROTOC_IMAGE=namely/protoc:1.23_0
```

Comment:
Can you mention somewhere why this changed?

Reply:
There's 3 small(?) changes in the generated files otherwise, I've noted the most interesting one in the commit message and above in the PR description